src/app/use-configure-query-client.js

```diff
@@ -1,14 +1,15 @@
 import { useDataEngine } from '@dhis2/app-runtime'
-import { QueryClient } from 'react-query'
+import { useQueryClient } from 'react-query'
 import { createWebStoragePersistor } from 'react-query/createWebStoragePersistor-experimental'
 import { persistQueryClient } from 'react-query/persistQueryClient-experimental'
 import { dataValueSetQuery } from '../data-workspace/data-workspace.js'
 
 // Persisted data will be garbage collected after this time
 const MAX_CACHE_AGE = 1000 * 60 * 60 * 24 * 31 // One month
 
-const useQueryClient = () => {
+const useConfigureQueryClient = () => {
```

Comment:
```suggestion
const useConfiguredQueryClient = () => {
```

Reply:
It's not yet already configured though, the configuration happens in this hook. Basically configureQueryClient, and then useConfigureQueryClient to comply with the rules of hooks.

Reply:
If it would return the configured queryClient I would prefer your name as well. Anyway, let me know what you think.

Reply:
Ah, sorry, I didn't get that.. My bad

Reply:
No worries, thanks for the review!